MIGRATING.md

```diff
@@ -2,11 +2,34 @@
 
 This guide explains what is needed to upgrade contracts when migrating over major releases of `sylvia`. Note that you can also view the [complete CHANGELOG](https://github.com/CosmWasm/sylvia/blob/main/CHANGELOG.md) to understand the differences.
 
+## 1.1.0 -> 1.2.0
 
-## 1.0.2 ->
+### Explicit generic customs in entry_points macro
 
+Sylvia can deduce the types used in place of generic customs.
+The `, custom(msg=..., query=...)` part is now not supported and can be safely removed.
+
+```diff
+-#[entry_points(generics<Empty, Empty>, custom(msg=Empty, query=Empty))]
++#[entry_points(generics<msg=Empty, query=Empty>)]
+#[contract]
+#[sv::custom(msg=E, query=Q)]
+impl<E, Q> CounterContract<E, Q>
+where
+    E: CustomMsg + 'static,
+    Q: CustomQuery + 'static,
+{
+}
+```
+
+### Removed `InterfaceApi` trait
+
+The `InterfaceApi` trait has been removed in favor of the `InterafceMessagesApi`.
```

Comment:
Let's add some example code here, because `InterfaceMessagesApi` lays in generated `sv` modules and `InterfaceApi` was a trait in `sylvia.types`